biz.aQute.bndlib/src/aQute/bnd/build/Project.java

```diff
@@ -2112,6 +2112,15 @@ public void test(File reportDir, List<String> tests) throws Exception {
 			IO.delete(reportDir);
 			tester.setReportDir(reportDir);
 		}
+		if (tests == null) {
+			// check system properties: gradle
+			String singleTest = System.getProperty("integTest.single");
```

Comment:
we can't use the globbing feature of gradle or maven at the plugin itself, because there are no classes there to filer. The class names are not available until WAY later during the test launcher framework execution. So we need a single form a globbing that is neither gradle nor maven supported anyway... so do we just check the system properties for the property bnd already defines (a.k.a. `tester.names`)?